README.md

```diff
@@ -3,6 +3,31 @@
 The repository provides a complete set of build tools for web developers. **These
 images MUST NOT be used in production**. The targeted usage of those images is GitlabCI.
 
+
+## Contributing
+
+To contribute you will need python3.6 and pipenv (installed by `pip install pipenv`).
+
+- Clone the repo
+- Create your pipenv environnement
+  > pipenv install
+- Load your pipenv
+  > pipenv shell
+- Run the script
+  > python travis build --image image --version version
+
+Script help:
```

Comment:
Can you directly update / move the "Testing" section (line 110 of this README) and provide a verbose example please ?

Reply:
I moved the "Testing" section to the beginning of the file.